epam-assets/theme/theme_electric.scss

```diff
@@ -320,7 +320,29 @@
         --uui-color-contrast: var(--night800);
     }
 
+    .uui-color-night100 {
+        --uui-color-5: var(--white);
+        --uui-color-10: var(--white);
+        --uui-color-20: var(--night50);
+        --uui-color-50: var(--night100);
+        --uui-color-60: var(--night200);
+        --uui-color-70: var(--night300);
+        --uui-color-contrast: var(--uui-neutral-80);
+    }
+
+    .uui-color-night300 {
+        --uui-color-5: var(--night50);
+        --uui-color-10: var(--night100);
+        --uui-color-20: var(--night200);
+        --uui-color-50: var(--night300);
+        --uui-color-60: var(--night400);
+        --uui-color-70: var(--night500);
+        --uui-color-contrast: var(--uui-neutral-80);
+    }
+
     .uui-color-neutral,
+    .uui-color-night600,
```

Comment:
we don't need 'night600' color in electric skin, because it's an alias for the semantic 'neutral' color and we decided that we will use semantic palette as is in electric

Reply:
fixed